### DISCUSSION TITLE
/api/v2/routing/otp/plan returning no itineraries?

### DISCUSSION THREAD

**Discussion by DaveNash** [transitland/transitland#427] [Q&A] [answered]
Various Bay Area agencies that returned results yesterday are returning empty arrays for itineraries today. For example: 
https://transit.land/api/v2/routing/otp/plan?fromPlace=37.75254,-122.19835&toPlace=37.79014,-122.39645&date=2024-06-05&time=08:54&arriveBy=true&maxWalkDistance=200&apikey=[apikey]

**Comment by irees** [comment]
Hi @DaveNash 
We've identified the problem and are working on a fix now. Should have an update later today.

**Comment by DaveNash** [comment]
Thanks, Ian. Nice to hear from you. Hope you’re well.

**Reply by irees** [reply, marked as answer]
It should be resolved now!

**Comment by DaveNash** [comment]
Yep, thanks, Ian. 

One follow-up though; is it me, or does the routing api return two of the same itinerary every time? 
I could be confused by my own post-processing, but I’m fairly certain the following returns each itinerary twice, for example: 
https://transit.land/api/v2/routing/otp/plan?fromPlace=37.78636,-122.39817&toPlace=37.5108,-122.2717&date=2024-06-12&time=17:00&arriveBy=false&apikey=3alXYEvBRw4SbYvQ3SwydamT3XMqRe6F&maxWalkDistance=10867&allowWalkingItinerary=true

Dave

**Comment by DaveNash** [comment]
This API is timing out again. ???

"message": "The upstream server is timing out"

**Comment by DaveNash** [comment]
Working again; seems faster, too. Thanks
